applications/ecallmgr/src/ecallmgr_fs_authz.erl

```diff
@@ -83,6 +84,13 @@ maybe_authorize_channel(Props, Node) ->
             end
     end.
 
+-spec is_referred(wh_proplist()) -> 'true' | 'false'.
+is_referred(Props) ->
+  case kzd_freeswitch:ccv(Props, [<<"Referred-By">>, <<"Referred-To">>]) of
```

Comment:
Aah I see, I thought kzd_freeswitch:ccv/2 is passing Key as is. I’ll fix it.

From: James Aimonetti <notifications@github.com<mailto:notifications@github.com>>
Reply-To: 2600hz/kazoo <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, December 14, 2015 at 3:10 PM
To: 2600hz/kazoo <kazoo@noreply.github.com<mailto:kazoo@noreply.github.com>>
Cc: Hesaam Farhang <hesaam@2600hz.com<mailto:hesaam@2600hz.com>>
Subject: Re: [kazoo] KAZOO 4436 (#1487)


In applications/ecallmgr/src/ecallmgr_fs_authz.erl<https://github.com/2600hz/kazoo/pull/1487#discussion_r47573858>:

> @@ -83,6 +84,13 @@ maybe_authorize_channel(Props, Node) ->
>              end
>      end.
>
> +-spec is_referred(wh_proplist()) -> 'true' | 'false'.
> +is_referred(Props) ->
> +  case kzd_freeswitch:ccv(Props, [<<"Referred-By">>, <<"Referred-To">>]) of


I'm confused how this can ever return a two-tuple? Did you mean to put those in separate calls to kzd_freeswitch:ccv/2?

—
Reply to this email directly or view it on GitHub<https://github.com/2600hz/kazoo/pull/1487/files#r47573858>.